ratis-grpc/src/main/java/org/apache/ratis/grpc/server/GrpcLogAppender.java

```diff
@@ -591,38 +591,40 @@ private TermIndex shouldNotifyToInstallSnapshot() {
     final FollowerInfo follower = getFollower();
     final long leaderNextIndex = getRaftLog().getNextIndex();
     final boolean isFollowerBootstrapping = getLeaderState().isFollowerBootstrapping(follower);
-
+    long leaderStartIndex = getRaftLog().getStartIndex();
+    TermIndex leaderStartTermIndex = getRaftLog().getTermIndex(leaderStartIndex);
+    SnapshotInfo snapshot = getServer().getStateMachine().getLatestSnapshot();
```

Comment:
We don't really need to get the snapshot since it is only used for the precondition check.